simple/php/rename-mehod_before.php

```diff
@@ -0,0 +1,13 @@
+class Customer {
+  $name = "name";
```

Comment:
Make the fields private here and in rest of the versions. This would make more sense.